src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -407,7 +421,7 @@ export abstract class AbstractListSettingWidget<TDataItem extends object> extend
 		rowElement.title = this.getLocalizedRowTitle(item);
 		rowElement.setAttribute('aria-label', rowElement.title);
 
-		if (item.selected && listFocused) {
+		if (!this.model.rendersEditOnly && item.selected && listFocused) {
```

Comment:
It seems like this rendersEditOnly property would be a property of the bool object renderer, not of a model, since this is just about how the rows should be displayed.